app/templates/signup.html

```diff
@@ -2,18 +2,25 @@
 
 {% block content %}
 
-<h1>Sign Up & Register</h1>
+<html>
```

Comment:
This template should not have the `html` qualifier/tag since it is placed in a block content in the page